src/components/header/Nav.js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import SearchBar from "./SearchBar";
+
+const Nav = ({ fetchMoviesData }) => {
+  const [isNavBlack, setIsNavBlack] = useState(false);
+
+  //this function update state by checking the scrollY position
+  const transitionNavbar = () => {

```

Comment:
LOVE the use of navBar transition and stick to the top when user scroll down. Great user experience!